### DISCUSSION TITLE
404: NOT_FOUND Code: DEPLOYMENT_NOT_FOUND

### DISCUSSION THREAD

**Discussion by larry840** [vercel/community#6460] [Help] [answered]
### Summary
https://project-5-snake-game.vercel.app/
It shows 
404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: hkg1::4bqxq-1712736368789-069023b52fac

It is a very simple website, just includes one html, one javascript, one css file.
https://github.com/larry840/project-5-snake-game
Please help me to solve this, thank you so much!

### Steps to Reproduce
https://project-5-snake-game.vercel.app/
It shows 
404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: hkg1::4bqxq-1712736368789-069023b52fac

It is a very simple website, just includes one html, one javascript, one css file.
https://github.com/larry840/project-5-snake-game
Please help me to solve this, thank you so much!

**Comment by amyegan** [comment, marked as answer]
Hey @larry840. Please rename `Index.html` to lowercase `index.html` and let me know if that works for you 🙏

**Reply by larry840** [reply]
It works! Thank you so much! How stupid I am🫣

**Reply by amyegan** [reply]
Not stupid at all! I'm just glad the solution was an easy one 😄

**Comment by GinaCastromonte** [comment]
Hi @amyegan  I got a similar error for a project I deleted and it seems like the domain is still attached.  The domain in question is www.runaboutfarm.org .  I deleted the project and removed the domain from my account.

**Reply by amyegan** [reply]
Hey @GinaCastromonte. Have you changed the DNS settings or connected the domain to a different project? If not, this seems like expected behavior since the domain wouldn't have been given instruction to point anywhere else yet.

This is slightly different issue from the one originally reported here, so please [start a new discussion](https://github.com/vercel/community/discussions/new) if you need more help. Be sure to include as much info as possible about what is happening and what you've already tried. And feel free to tag me there to make sure I see it!

**Comment by GinaCastromonte** [comment]
It is connected to Homestead but somehow Vercel's error message still keeps showing up.

**Reply by amyegan** [reply]
It looks like the domain is still pointing to Vercel https://www.whatsmydns.net/#CNAME/www.runaboutfarm.org

If you need more help, please [start a new discussion](https://github.com/vercel/community/discussions/new) 🙏
